tsconfig.json

```diff
@@ -1,17 +1,35 @@
 {
   "extends": "./node_modules/adonis-preset-ts/tsconfig",
-  "include": ["**/*"],
-  "exclude": ["node_modules", "build"],
+  "include": [
+    "**/*"
+  ],
+  "exclude": [
+    "node_modules",
+    "build"
+  ],
   "compilerOptions": {
     "outDir": "build",
     "rootDir": "./",
     "sourceMap": true,
     "paths": {
-      "App/*": ["./app/*"],
-      "Config/*": ["./config/*"],
-      "Contracts/*": ["./contracts/*"],
-      "Database/*": ["./database/*"]
+      "App/*": [
```

Comment:
Any reason you want to use capitalized letters on this?

Reply:
Those are default settings for adonisjs paths for the project when one does a fresh installation and in most cases, the tsconfig.json file is untouched throughout the development process.

Reply:
Okoko!